package.json

```diff
@@ -4,17 +4,22 @@
   "description": "JavaScript API client library for Percy (https://percy.io).",
   "main": "dist/main.js",
   "scripts": {
-    "prepublish": "npm run build",
-    "build:compile": "babel src --presets babel-preset-es2015 --out-dir dist",
-    "build:browserify": "browserify --standalone percy-client dist/main.js -o dist/bundle.js",
     "build": "npm run build:compile && npm run build:browserify",
-    "test": "mocha --require babel-core/register",
+    "build:browserify":
+      "browserify --standalone percy-client dist/main.js -o dist/bundle.js",
+    "build:compile": "babel src --presets babel-preset-es2015 --out-dir dist",
     "debug": "mocha debug --require babel-core/register",
-    "tdd": "mocha --require babel-core/register --watch"
+    "lint": "./node_modules/eslint/bin/eslint.js .",
+    "lint:fix": "./node_modules/eslint/bin/eslint.js . --fix",
```

Comment:
`lint` and `lint:fix` are the only two script changes.  All the others are simply re-ordered.